src/main/java/graphql/validation/Validator.java

```diff
@@ -27,7 +27,7 @@
 import graphql.validation.rules.PossibleFragmentSpreads;
 import graphql.validation.rules.ProvidedNonNullArguments;
 import graphql.validation.rules.ScalarLeaves;
-import graphql.validation.rules.SubscriptionUniqueRootField;
+import graphql.validation.rules.SubscriptionRootField;
```

Comment:
I changed this rule name to be more generic, reflecting that it does 2 checks on the root field